docker-entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash --login
+# NOTE: By using --login, it guarantees /etc/profile is always sourced, unlike the
+# non-login, non-interactive shell you get by default with `docker run`.
+
+set -e
+
+echo "Using '$(python --version 2>&1)' from '$(which python)'"
+
+# FIXME: Could we replace this with a Docker Compose env_file?
+# SEE: https://docs.docker.com/compose/compose-file/#env_file
+if [ -f "${APP_HOME}/.env" ]; then
```

Comment:
This doesn't seem to work properly for me. I don't have an `APP_HOME` environment variable in the container. Does the use of [`ENV`](https://docs.docker.com/engine/reference/builder/#env) in the Dockerfile actually persist at container runtime?

Reply:
I believe APP_HOME is only set in cfgov-base and what's running via compose is cfgov-develop.

Reply:
Then again, I have APP_HOME set after `docker-compose exec python2 bash --login`

Reply:
I should have specified, I followed the [documentation](https://cfpb.github.io/cfgov-refresh/running-docker/#access-a-containers-shell) that suggests using `docker exec -it cfgov-refresh_python2_1 bash`.